README.md

```diff
@@ -9,21 +9,59 @@ server is deployed at:
 
 https://go-acd.appspot.com
 
-# Install
+# Usage
+
+In order to use this library, you must authenticate through the token server. A  (e.g. https://go-acd.appspot.com).
 
-This project is go-gettable:
+Get and build:
 
 ```
-go get gopkg.in/acd.v0/...
+$ go get gopkg.in/acd.v0/cmd/acd
+$ go build gopkg.in/acd.v0/cmd/acd
```

Comment:
@dsoprea do you want to make this change so I can merge? I'll also take a look at the CI, I believe it's failing because the secrets do not exist.